mailpoet/lib/EmailEditor/Integrations/MailPoet/Patterns/PatternsController.php

```
@@ -140,7 +140,7 @@ public function registerPatterns(): void {
         continue;
       }
 
-      register_block_pattern($patternData['name'], $patternData['properties']);
+      register_block_pattern($patternData['name'], $patternData['properties']); // @phpstan-ignore argument.type (validated as array<string, mixed> just above; register_block_pattern's strict shape is enforced at runtime)
```

PHPStan (2.1.51)

[error] No error with identifier argument.type is reported on line 143.


(ignore.unmatchedIdentifier)